GVRf/Sample/gvr-test-screenshot3Dresult/src/org/gearvrf/testscreenshot3Dresult/TestScreenshot3DResultScript.java

```diff
@@ -0,0 +1,99 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.testscreenshot3Dresult;
+
+import org.gearvrf.GVRAndroidResource;
+import org.gearvrf.GVRContext;
+import org.gearvrf.testscreenshot3Dresult.R;
+import org.gearvrf.FutureWrapper;
+import org.gearvrf.GVRMesh;
+import org.gearvrf.GVRScene;
+import org.gearvrf.GVRSceneObject;
+import org.gearvrf.GVRScript;
+
+import android.util.Log;
+
+public class TestScreenshot3DResultScript extends GVRScript {
+
+    private static final float CUBE_WIDTH = 20.0f;
+    private GVRContext mGVRContext = null;
+
+    @Override
+    public void onInit(GVRContext gvrContext) {
+        mGVRContext = gvrContext;
+
+        GVRScene scene = mGVRContext.getNextMainScene();
+
+        FutureWrapper<GVRMesh> future_mesh = new FutureWrapper<GVRMesh>(
```

Comment:
The modified version of "gvrample" shows how to use the new API to capture 3D screenshot, and save the result into six image files.

It is difficult for people (particularly reviewers) to tell whether the six images generated above are correct or not (i.e. whether they connect to each other perfectly) by simply view them by eyes. So I create the second sample "gvrtestscreenshot3Dresult". In this sample, I create a cube and map the six images to its faces. So the user can easily tell whether the result of 3D screenshot is correct or not. This sample is mainly for review reason. We can delete it after this commit is reviewed and merged.